app/src/main/java/com/example/game/activities/LoginActivity.java

```diff
@@ -0,0 +1,76 @@
+package com.example.game.activities;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.example.game.R;
+import com.example.game.databinding.ActivityLoginBinding;
+import com.google.android.material.snackbar.BaseTransientBottomBar;
+import com.google.android.material.snackbar.Snackbar;
+import com.parse.LogInCallback;
+import com.parse.ParseException;
+import com.parse.ParseUser;
+
+public class LoginActivity extends AppCompatActivity {
+    private ActivityLoginBinding binding;
+    private EditText etEmail;
+    private EditText etPassword;
+    private Button btnLogin;
+
+    public static final String  TAG = "LoginActivity";
```

Comment:
```suggestion
    public static final String  TAG = "LoginActivity";
    
    private ActivityLoginBinding binding;
    private EditText etEmail;
    private EditText etPassword;
    private Button btnLogin;
```
Refer to java code style of parameter order.